static/js/years.js

```diff
@@ -0,0 +1 @@
+var ACCYEARS = {"\u05e9\u05e0\u05ea 2013": ["01/01/2013", "31/12/2013"]};
```

Comment:
Will be changed when `process.py` sruns on the server.
This is just the result I had using the example .csv